examples/mongodb/0.6/init-inventory.sh

```diff
@@ -0,0 +1,72 @@
+HOSTNAME=`hostname`
+
+echo "Waiting for MongoDB to start"
+
+sleep 10
+mongo localhost:27017/inventory <<-EOF
+    rs.initiate({
+        _id: "rs0",
+        members: [ { _id: 0, host: "${HOSTNAME}:27017" } ]
```

Comment:
Maybe it should be changed to
```
members: [ { _id: 0, host: "localhost:27017" } ]
```

Reply:
That's what I originally had, but then it fails apart when registering the DBZ connector, as it cannot find MongoDB on `localhost`. I assume the Mongo client receives all the cluster members and tries to access them. Of course there's no Mongo running on `localhost` on the Connect node.

I'm kinda stuck here, when I use the actual host name in the call to `rs.initiate()` that request can't be resolved when that initialization script is run. I reckon that's because the host name doesn't yet exist in `etc/hosts` when that script is run. The only way I see right now is to let the user run that initialisation script themselves after everything has been started, but that's kinda ugly. Do you have any other idea?

Reply:
Btw. I had worked around that by adding

    extra_hosts:
    - "mongodb:127.0.0.1"

To the MongoDB service in my Compose file, but I agree that the Docker image itself should be usable without any out-of-band knowledge.